environment.yml

```diff
@@ -9,4 +9,4 @@ dependencies:
   - conda-standalone =4.12a=*_9
   - menuinst =2.0a=14_g594d642_*
   - conda =4.12a
-  - mamba
+  - mamba =1.1.0
```

Comment:
Perhaps we need to pin more than this... I think this was the `mamba` version that was available on Dec 23rd, which was our last successful build.